README.md

```diff
@@ -1,3 +1,53 @@
-# Med API
+<section style="text-align: center;">
 
-TBD
+# Federation of State Medical Boards</br>MED API Documentation
+</section>
+
+This is the source for technical information for the Federation of State Medical Boards' Federation Credentials Verification Service (MED) RESTful API. 
+The MED API is one of the interfaces available for MED data retrieval by state medical boards (SMBs). 
```

Comment:
This isn't descriptive to me. If this came from the docs then leave it and I'll talk with Cyndi about it.